Update offboarding.md (#505)
mattermost/mattermost-handbook: operations/workplace/people/offboarding.md

# Offboarding 

## Instructions for Managers

> 1. Forward resignation letter/email to MLT member and HR hr@mattermost.com. The resignation letter/email will be stored in Bamboo.

Reviewer: ```suggestion
1. Forward staff member's resignation letter/email to MLT member and HR hr@mattermost.com. The resignation letter/email will be stored in BambooHR.
```

> Note: a resignation letter/email is required from departing staff. In most cases and depending upon the statutory requirements in the staff member’s country/region, a minimum of two weeks' notice is required for voluntary departures.

Reviewer: ```suggestion
**Note:** A resignation letter/email is required from departing staff. In most cases and depending upon the statutory requirements in the staff member’s country/region, a minimum of two weeks' notice is required for voluntary departures.
```

2. Complete the [Manager Offboarding Checklist](https://form.asana.com?hash=1a70cad9b3facd446307bb0209313f9909a1f2a8a035b5bc65b92e738b0d3057&id=1177889065987940) which is used to indicate:
  * Systems that need to be deprovisioned.

>   * Confirm whether emails need to be forwarded to the member/selected team member for a period following staff member’s departure.

Reviewer: ```suggestion
  * Confirm whether emails need to be forwarded to a manager/selected team member for a period following staff member’s departure.
```

>   * Confirm if there are any company-owned files in Gdrive that need to be transferred to you.

Reviewer: ```suggestion
  * Confirm if there are any company-owned files in Google Drive that need to be transferred to you.
```

  * Review and approve any outstanding expense reports.

> 3. If departing team member is a [CAO](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#cao) for any vendors, systems or contractors, communicate changes to Finance regarding: 

Reviewer: ```suggestion
3. If the departing team member is a [CAO](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#cao) for any vendors, systems, or contractors, communicate changes to Finance regarding:
```

  * Which vendors need to be transitioned (billing approvals, main point of contact).

>   * Confirm status on open procurement requests and transition procurement DRI via procurement@mattermost.com. Include: most recent email exchange and new vendor DRI.

Reviewer: ```suggestion
  * Status confirmation on open procurement requests and transition procurement DRI via procurement@mattermost.com. Include: most recent email exchange and new vendor DRI.
```

  * Transfer ownership of any software accounts to Finance or a different team DRI. Notify [Systems channel](https://community.mattermost.com/private-core/channels/systems) of the change.

> 4. Confirm [Communication Plan](https://handbook.mattermost.com/operations/workplace/people/offboarding#communicating-departures) with HR. Managers may also 

Reviewer: ```suggestion
4. Confirm [Communication Plan](https://handbook.mattermost.com/operations/workplace/people/offboarding#communicating-departures) with HR. Managers may also share:
```

> * Staff member’s termination details, including: date, final vacation balance ([if located in Canada, UK or Germany](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#when-you-take-time-off-full-days-only-please-do-the-following)) 

Reviewer: ```suggestion
 * Staff member’s termination details, including: date, final vacation balance ([if located in Canada, UK, or Germany](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#when-you-take-time-off-full-days-only-please-do-the-following)).
```

  * If needed, provide context on what led to the staff member’s resignation.
  * Confirm who will be taking over staff member’s responsibilities in the interim.

> 5. Termination Retrospect with HR or Team MLT. This is not a formal [retrospect](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-do-a-post-escalation-post-mortem), but is an opportunity to understand how to improve as a function, team and company. HR may initiate this during Step 3, and will also request Manager feedback on:  

Reviewer: ```suggestion
5. Termination Retrospective with HR or Team MLT. This is not a formal [retrospective](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-do-a-post-escalation-post-mortem), but is an opportunity to understand how to improve as a function, team ,and company. HR may initiate this during Step 3, and will also request Manager feedback on:
```

  * Confirm rehire eligibility (if resignation was regrettable, not regrettable).

>   * Confirm whether a backfill is required. If yes, follow instructions to [open a new role](https://handbook.mattermost.com/contributors/join-us/staff-recruiting#how-to-open-a-new-role). 

Reviewer: ```suggestion
  * Confirm whether a backfill is required. If yes, follow instructions to [open a new role](https://handbook.mattermost.com/contributors/join-us/staff-recruiting#how-to-open-a-new-role).
```

> Thank the departing staff member for their contributions and wish them the best for the future. Every person that joins Mattermost is part of the team and welcome to join our [Community](https://mattermost.com/community/) and stay in touch in whatever capacity feels most comfortable.  

Reviewer: ```suggestion
Thank the departing staff member for their contributions and wish them the best for the future. Every person that joins Mattermost is part of the team and welcome to join our [Community](https://mattermost.com/community/) and stay in touch in whatever capacity feels most comfortable.
```

HR will reach out to the departing staff member to communicate last day instructions and details and to schedule an [exit interview](https://docs.google.com/document/d/1MX6LXa2uVEM3_xqH-pqKyJH4PWis9oJ0a5sAHd58W1U/edit?ts=5e99ecc0).

## Instructions for Departing Team Members

> 1. Forward resignation letter/email to MLT member and HR hr@mattermost.com. The resignation letter/email will be stored in Bamboo.

Reviewer: ```suggestion
1. Forward your resignation letter/email to MLT member and HR hr@mattermost.com. The resignation letter/email will be stored in BambooHR.
```

> Note: a resignation letter/email is required from departing staff. In most cases and depending upon the statutory requirements in the staff member’s country/region, a minimum of two weeks' notice is required for voluntary departures.

Reviewer: ```suggestion
**Note:** A resignation letter/email is required from departing staff. In most cases and depending upon the statutory requirements in the staff member’s country/region, a minimum of two weeks' notice is required for voluntary departures.
```

> 2. Work with your Manager to develop a [transition plan]([Communication Plan](https://handbook.mattermost.com/operations/workplace/people/offboarding#plan-for-transition) and [Communication Timeline](https://handbook.mattermost.com/operations/workplace/people/offboarding#communicating-departures). Departing Team Members should feel encouraged to say goodbye to peers,  We encourage team members to [make meaninful connections](https://handbook.mattermost.com/contributors/join-us/recruiting#4-make-meaningful-connections) and encourage departing team members to say goodbye to peers, or optionally post in the [Stand-up Channel](https://community.mattermost.com/private-core/channels/stand-up). We just ask to align with your Manager on timeline of messaging so that direct teammates or working groups can be informed with an adequate plan on  

Reviewer: ```suggestion
2. Work with your Manager to develop a [transition plan]([Communication Plan](https://handbook.mattermost.com/operations/workplace/people/offboarding#plan-for-transition) and [Communication Timeline](https://handbook.mattermost.com/operations/workplace/people/offboarding#communicating-departures). We encourage team members to [make meaningful connections](https://handbook.mattermost.com/contributors/join-us/recruiting#4-make-meaningful-connections) and encourage departing team members to say goodbye to peers, or optionally post in the [Stand-up Channel](https://community.mattermost.com/private-core/channels/stand-up). We just ask to align with your Manager on timeline of messaging so that direct teammates or working groups can be informed with an adequate plan ahead of departure.
```

> 3. Share Feedback - optional. Input is highly encouraged and helps People understand [likes and wishes](https://handbook.mattermost.com/company/about-mattermost/mindsets#likes-and-wishes) as it relates to your experience at Mattermost and helps identify opportunities to improve.  

Reviewer: ```suggestion
3. Share feedback (optional). Input is highly encouraged and helps the People team understand [likes and wishes](https://handbook.mattermost.com/company/about-mattermost/mindsets#likes-and-wishes) as they relate to your experience at Mattermost and helps identify opportunities to improve.
```

>   * **Exit Survey.** All departing staff will recieve an Exit Survey feedback request through CultureAmp. The survey link will be sent via email after the termination date is recorded in BambooHR.  

Reviewer: ```suggestion
  * **Exit Survey:** All departing staff will receive an Exit Survey feedback request through CultureAmp. The survey link will be sent via email after the termination date is recorded in BambooHR.
```

>   * **Exit Interview with HR.** Staff who leave on a voluntary basis will be invited to share feedback on their Mattermost experience in an exit interview with HR. The results of the exit interview will be shared with the departing team member’s manager and MLT. You may keep all or parts of your feedback confidential - as long as it does not involve matters that legally require additional action by the HR team.  While exit interviews are not mandatory, we greatly appreciate your feedback about what is going well and what can be improved to continue making Mattermost a great place to work. 

Reviewer: ```suggestion
  * **Exit interview with HR:** Staff who leave on a voluntary basis will be invited to share feedback on their Mattermost experience in an exit interview with HR. The results of the exit interview will be shared with the departing team member’s manager and MLT. You may keep all or parts of your feedback confidential - as long as it does not involve matters that legally require additional action by the HR team. While exit interviews are not mandatory, we greatly appreciate your feedback about what is going well and what can be improved to continue making Mattermost a great place to work.
```

> 4. Return Equipment. All Mattermost property needs to be returned. This includes laptops, tablets, test devices, and any Intellectual Property. Complete the [Mattermost Devices Return Form](https://docs.google.com/forms/d/1rzHRsODVWtNaic3-T8DidTziUKtoLNo34qOlOV9i5QI/viewform?edit_requested=true). Hanna Park will send you a prepaid shipping label to coordinate your equipment return. 

Reviewer: ```suggestion
4. Return equipment. All Mattermost property needs to be returned. This includes laptops, tablets, test devices, and any Intellectual Property. Complete the [Mattermost Devices Return Form](https://docs.google.com/forms/d/1rzHRsODVWtNaic3-T8DidTziUKtoLNo34qOlOV9i5QI/viewform?edit_requested=true). Hanna Park will send you a prepaid shipping label to coordinate your equipment return.
```

> Note: you do not need to and you will not be asked to return home office equipment, such as: desk, chair, laptop stand, etc.  

Reviewer: ```suggestion
**Note:** You do not need to, and you will not be asked to, return home office equipment, such as: desk, chair, laptop stand, etc.
```

> 5. Stay in touch. Thank you for your time at Mattermost. Every person that joins Mattermost is part of the team and welcome to join our [Community](https://mattermost.com/community/) and stay in touch in whatever capacity feels most comfortable.  

Reviewer: ```suggestion
5. Stay in touch. Thank you for your time at Mattermost. Every person that joins Mattermost is part of the team and welcome to join our [Community](https://mattermost.com/community/) and stay in touch in whatever capacity feels most comfortable.
```

## Communicating Departures

Work with the departing staff member on the messaging around their departure. Confirm timeline of messaging and request that the staff member refrain from sharing info ahead of the planned timeline.

Communicate the agreed-upon messaging around the departure with the relevant team, department, and any other appropriate groups as soon as possible. This is important to reinforce asynchronous communication. Silence and/or lack of communication can lead to confusion, misalignment, conjecture, gossip, and insecurity. Include the departing staff member in the announcement if they are comfortable participating in the discussion.

To avoid “@ all” departure emails/Mattermost messages, invite the departing staff member to draft a message to be shared in the [Stand-Up Channel](https://community.mattermost.com/private-core/channels/stand-up). This message should be reviewed by the departing staff member's manager and HR.

## Plan for the Transition

> Managers should ensure that a knowledge transfer plan is in place with the departing staff membe, this may include:

Reviewer: ```suggestion
Managers should ensure that a knowledge transfer plan is in place with the departing staff member. This may include:
```
